web-admin/src/features/organizations/user-management/table/groups/GroupCompositeCell.svelte

```diff
@@ -23,10 +46,48 @@
     <span class="text-sm font-medium text-gray-900 flex flex-row gap-x-1">
       {name}
     </span>
-    <div class="flex flex-row items-center gap-x-1">
-      <span class="text-xs text-gray-500">
-        {usersCount} user{usersCount > 1 ? "s" : ""}
-      </span>
-    </div>
+    <Tooltip location="right" alignment="start" distance={8}>
+      <div
+        class="flex flex-row items-center gap-x-1"
+        role="button"
+        tabindex="0"
+        on:mouseenter={() => (hovered = true)}
+        on:focus={() => (hovered = true)}
+        on:blur={() => (hovered = false)}
+      >
+        <span class="text-xs text-gray-500">
+          {usersCount} user{usersCount === 1 ? "" : "s"}
+        </span>
+      </div>
+      <TooltipContent slot="tooltip-content">
+        {#if (usersCount ?? 0) === 0}
+          <div class="text-xs text-gray-300 px-1 py-0.5">No users</div>
+        {:else if $listUsergroupMemberUsers.isLoading}
+          <div class="text-xs text-gray-300 px-1 py-0.5">Loading…</div>
+        {:else if $listUsergroupMemberUsers.isError}
+          <div class="text-xs text-red-300 px-1 py-0.5">
+            Failed to load users
+          </div>
+        {:else}
+          <ul>
+            {#each previewUsers.slice(0, PREVIEW_COUNT) as u}
+              <div class="flex items-center gap-1 py-1">
+                <Avatar
+                  src={u.userPhotoUrl}
+                  avatarSize="h-4 w-4"
+                  fontSize="text-[10px]"
+                  alt={u.userName || u.userEmail}
+                  bgColor={getRandomBgColor(u.userEmail || u.userName)}
```

Comment:
Why is the order different here vs the alt and label? If they are supposed to be the same then use a variable for the 3 occurrences. If not then please add a comment stating why this is different.